### PR TITLE
Bump version to 0.6.6

### PR DESCRIPTION
**Pull request by umezawatakeshi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [0.6.6] - 2021-12-10
+
+### Changed
+
+- update supported k8s version to 1.22 (#75)
```
Comment (masa213f) [inline CHANGELOG.md:12]: It is good to write the license change.
https://github.com/cybozu-go/contour-plus/pull/73

Comment (umezawatakeshi) [reply CHANGELOG.md:12]: I don't care write it or not, but other software don't state license change in CHANGELOG.

What do you think?

Comment (masa213f) [reply CHANGELOG.md:12]: In my opinion, the license change affects the users. So it is better to write it.

**Review by masa213f — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,17 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+  - kubernetes 1.22.1
+  - contour 1.19.1
+  - cert-manager 1.6.1
+  - external-dns 0.10.1
+- Change LICENSE from MIT to Apache 2.0 (#73)
```
Comment (umezawatakeshi) [inline CHANGELOG.md:17]: It's written below.

**Pull request merged**